lib/msal-browser/src/interaction_client/SilentIframeClient.ts

```diff
@@ -59,13 +61,22 @@ export class SilentIframeClient extends StandardInteractionClient {
             // Create authorize request url
             const navigateUrl = await authClient.getAuthCodeUrl(silentRequest);
 
-            return await this.silentTokenHelper(navigateUrl, authCodeRequest, authClient, this.logger);
+            return await this.silentTokenHelper(navigateUrl, authCodeRequest, authClient, this.logger)
+                .then((result: AuthenticationResult) => {
+                    acquireTokenMeasurement.endMeasurement({
+                        success: true
```

Comment:
Do we need to set the fromCache here?

Reply:
We can, arguably superfluous though, given SilentIframeClient never reads from the cache.